ion-schema/src/isl/mod.rs

```diff
@@ -730,338 +716,6 @@ mod isl_tests {
         values.iter().cloned().map(|v| v.into()).collect()
     }
 
-    #[rstest(
```

Comment:
🗺️  These tests are for ranges, so when I rewrote the Range implementation, I moved/ported them from here to the new `ranges` module.